tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/service/TargetPlatformDefaultValueService.java

```diff
@@ -32,7 +32,7 @@
 
 	@Override
 	protected String compute() {
-		String[] liferayTargetPlatformVersions = WorkspaceConstants.liferayTargetPlatformVersions.get("7.2");
+		String[] liferayTargetPlatformVersions = WorkspaceConstants.liferayTargetPlatformVersions.get("7.3");
```

Comment:
Lets leave the default as 7.2 for now.  We can create a ticket to move the default to 7.3 in the future, but for now I don't think 7.3 is stable enough for us to make it the default.

Reply:
ok, I will revert it.